src/components/_modal/pages/pickup/PickupSelection.js

```diff
@@ -15,17 +15,16 @@ import { handleOnSelect } from "@/components/_modal/utils";
  * Special component responsible for loading order policy
  * Will not render anything, but needs to be mounted
  */
-function PolicyLoader({ branch, onLoad, pid, requireDigitalAccess }) {
+function PolicyLoader({ branch, onLoad, pid, pids, requireDigitalAccess }) {
```

Comment:
burde de ikke også give pids med i MultiOrderCheckoutForm.js når der pushes pickup modalen? omkring linje 116?

Reply:
jeg troede at hele finten var at multiordercheckoutform netpu IKKE skulle lave orderpolicy check - det giver ihvertfald ingen mening for mig at se

Reply:
ah, du har ret, det sker jo i materialet. glem den her så 👍